### PR TITLE
Update Teleport Cloud FAQ

### PR DESCRIPTION
**Pull request by wadells**
These topics come up frequently in our internal support channel.
    
Contributes to https://github.com/gravitational/cloud/issues/1297

* 2e4c6615561d9b3efab03b777e73cca49c6619ed has the notable changes. 
* 43349248400bea28f895660b9a7b02887d8c6167 is cleanup.

## Testing Done
Checked that the eventhandler link resolved correctly at:

https://github.com/gravitational/teleport/blob/walt/cloud-faq/docs/pages/cloud/faq.mdx

### CODE REVIEW

File: docs/pages/cloud/faq.mdx [hunk 1]
```diff
@@ -150,6 +154,10 @@ Proxy recording mode is disabled for cloud customers
 
 The ability to download recordings for offline viewing will be available in a future release.
 
+## Is there a way to forward Teleport Cloud audit events to my companies internal SEIM?
```
Comment (ulysseskan) [inline docs/pages/cloud/faq.mdx:156]: I think this should be `company's`.  And probably `SIEM` should be spelled out too along with the acronym.

**Review by stevenGravy — COMMENTED** [review]
For the question

## How can I access the tctl admin tool?

Can we add the link to the downloads page in cloud section?

File: docs/pages/cloud/faq.mdx (hunk 1 above)
Comment (wadells) [reply docs/pages/cloud/faq.mdx]: Changed both of these in f0a79a209cb3678b0de09d35986ae2ce740001d.  Thanks!

**Comment by wadells** [comment]
> Can we add the link to the downloads page in cloud section?

Fixed in [b82410b](https://github.com/gravitational/teleport/pull/12496/commits/b82410b3bf9f79b629953afbf26a27704da359be)

**Review by stevenGravy — APPROVED** [review]
lgtm

File: docs/pages/cloud/faq.mdx [hunk 2]
```diff
@@ -150,6 +158,10 @@ Proxy recording mode is disabled for cloud customers
 
 The ability to download recordings for offline viewing will be available in a future release.
 
+## Is there a way to forward Teleport Cloud audit events to my company's internal Security Event and Incident Managment (SEIM)?
```
Comment (webvictim) [inline docs/pages/cloud/faq.mdx:161]: I traditionally see "SIEM" used more than "SEIM", but maybe I'm behind the times?

**Review by webvictim — APPROVED** [review]

File: docs/pages/cloud/faq.mdx (hunk 2 above)
Comment (wadells) [reply docs/pages/cloud/faq.mdx]: Good catch. Changed in [0e596c3](https://github.com/gravitational/teleport/pull/12496/commits/0e596c3bad29e200e8a27be85236450f7babebec).

**Review by ulysseskan — APPROVED** [review]

File: docs/pages/cloud/architecture.mdx [hunk 3]
```diff
@@ -21,11 +21,14 @@ Teleport Cloud Proxy does not terminate SSH sessions when using OpenSSH and `tsh
...
-able to set their data retention with us.
-
-Customers whose subscriptions lapse will have all session recordings, audit, and cluster state deleted between 7 and 30 days of the lapse.
+Data retention cannot currently be configured by customers. All Teleport Cloud
+customers have audit logs retained in DynamoDB for 1 year, cluster configuration
+retained in DynamoDB indefinitely, and session recordings retained in S3 indefinitely.
+When data retention policies are introduced, customers will be contacted and
+able to specify their preferred data retention schedules.
+
+Customers whose subscriptions lapse will have all session recordings, audit, and
```
Comment (ptgott) [inline docs/pages/cloud/architecture.mdx:30]:
```suggestion
Customers whose subscriptions lapse will have all session recordings, audit logs, and
```

File: docs/pages/cloud/architecture.mdx [hunk 4]
```diff
@@ -21,11 +21,14 @@ Teleport Cloud Proxy does not terminate SSH sessions when using OpenSSH and `tsh
...
+cluster state deleted between 7 and 30 days of the lapse.
```
Comment (ptgott) [inline docs/pages/cloud/architecture.mdx:31]:
```suggestion
cluster state deleted between 7 and 30 days after the lapse.
```

File: docs/pages/cloud/faq.mdx [hunk 5]
```diff
@@ -34,6 +38,10 @@ We use AWS-managed keys. Currently there is no option to provide your own key.
 It's a Teleport-managed S3 bucket with AWS-managed keys.
 Currently there is no way to provide your own bucket.
 
+## How long will Teleport Cloud retain my data?
+
+See our documentation on [data retention](./architecture.mdx#data-retention)
```
Comment (ptgott) [inline docs/pages/cloud/faq.mdx:43]:
```suggestion
See our documentation on [data retention](./architecture.mdx#data-retention).
```

File: docs/pages/cloud/faq.mdx [hunk 6]
```diff
@@ -43,14 +51,15 @@ There is no need to open any ports on your infrastructure for inbound traffic.
 
 ## How can I access the tctl admin tool?
 
-We have made changes to allow you to log into your cluster using `tsh`, then use `tctl` remotely:
+Find the appropriate download at [Teleport Cloud Downloads](./downloads.mdx). Use the enterprise version of `tctl` and `tsh`.
```
Comment (ptgott) [inline docs/pages/cloud/faq.mdx:54]:
```suggestion
Find the appropriate download at [Teleport Cloud Downloads](./downloads.mdx). Use the Enterprise version of `tctl`.
```

My understanding is that there is no separate Enterprise version of `tsh` ([source](https://github.com/gravitational/teleport/pull/10920#discussion_r848898440)).

File: docs/pages/cloud/faq.mdx [hunk 7]
```diff
@@ -43,14 +51,15 @@ There is no need to open any ports on your infrastructure for inbound traffic.
...
+
+After downloading the tools, first log into your cluster using `tsh`, then use `tctl` remotely:
```
Comment (ptgott) [inline docs/pages/cloud/faq.mdx:56]:
```suggestion
After downloading the tools, first log in to your cluster using `tsh`, then use `tctl` remotely:
```

File: docs/pages/cloud/faq.mdx [hunk 8]
```diff
@@ -70,18 +79,18 @@ After [connecting](#how-can-i-access-the-tctl-admin-tool) `tctl` to Teleport Clo
 $ tctl nodes add --ttl=5m --roles=node,proxy --token=$(uuid)
 ```
 
-## When will a security audit be available?
+## Is an independent security audit available?
 
-Security audit has been completed and is available [here](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/).
+A security audit has been completed and is available [here](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/).
```
Comment (ptgott) [inline docs/pages/cloud/faq.mdx:84]:
```suggestion
A security audit has been completed and is available at our [Audit Report 2021](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/) page.
```
Added a suggestion to make this a bit clearer (https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/#here-conceals-what-users-are-clicking).

File: docs/pages/cloud/faq.mdx [hunk 9]
```diff
@@ -70,18 +79,18 @@ After [connecting](#how-can-i-access-the-tctl-admin-tool) `tctl` to Teleport Clo
...
 
 ## Where does Teleport Cloud run?
 
-Teleport Cloud is deployed within AWS, using S3 and DynamoDB for storage with AWS managed encryption keys. All data is currently hosted in us-west-2. 
+Teleport Cloud is deployed within AWS, using S3 and DynamoDB for storage with AWS managed encryption keys. All data is currently hosted in us-west-2.
```
Comment (ptgott) [inline docs/pages/cloud/faq.mdx:88]:
```suggestion
Teleport Cloud is deployed within AWS, using S3 and DynamoDB for storage with AWS-managed encryption keys. All data is currently hosted in us-west-2.
```

**Review by ptgott — APPROVED** [review]
Approved with some minor suggestions

**Pull request merged**